src/components/Board/Output/SymbolOutput/ClearButton/ClearButton.js

```diff
@@ -27,16 +15,26 @@ export class ClearButton extends Component {
   };
 
   render() {
-    const { classes, hidden, ...other } = this.props;
+    const { classes, hidden, increaseOutputButtons, ...other } = this.props;
 
     return (
       <Scannable disabled={hidden}>
-        <IconButton aria-label="Clear" className={classes.button} {...other}>
-          <ClearIcon className={classes.icon} />
+        <IconButton
+          aria-label="Clear"
+          className={
+            increaseOutputButtons ? 'Output__button__lg' : 'Output__button__sm'
+          }
+          {...other}
+        >
+          <ClearIcon
+            className={
+              increaseOutputButtons ? 'Output__icon__lg' : 'Output__icon__sm'
+            }
+          />
         </IconButton>
       </Scannable>
     );
   }
 }
 
-export default withStyles(styles)(ClearButton);
+export default withStyles(null)(ClearButton);
```

Comment:
You can export directly the componente without the withStyles, like `export default (ClearButton)` and also deleate the import.